airbyte-integrations/connectors/source-redshift/src/main/java/io/airbyte/integrations/source/redshift/RedshiftSource.java

```diff
@@ -39,7 +44,20 @@ public JsonNode toDatabaseConfig(final JsonNode redshiftConfig) {
             redshiftConfig.get("host").asText(),
             redshiftConfig.get("port").asText(),
             redshiftConfig.get("database").asText()));
+
+    if (redshiftConfig.has(SCHEMAS) && redshiftConfig.get(SCHEMAS).isArray()) {
+      schemas = new ArrayList<>();
```

Comment:
nitpick: you can use guava's `Lists.newArrayList(redshiftConfig.get(SCHEMAS))` instead of manually looping.

Reply:
Using the guava node we get a list of JsonNode:
ArrayList<JsonNode> jsonNodes = Lists.newArrayList(redshiftConfig.get(SCHEMAS));

But we need to get a list of Sring (get strings value from JsonNodes) then take a string value from this node. So it seems like anyway some kind of iteration would be required ( loop or stream).
Or am I missing something?

Reply:
ah my bad, I missed that. This makes sense.